docs/resources/User.md

```diff
@@ -24,23 +24,23 @@ There are other rules and restrictions not shared here for the sake of spam and
-| public_flags? | integer   | the public [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                  | identify              |
+| Field         | Type      | Description                                                                                                                       |
+| ------------- | --------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| id            | snowflake | the user's id. Requires `identify` scope                                                                                          |
+| username      | string    | the user's username, not unique across the platform. Requires `identify` scope                                                    |
+| discriminator | string    | the user's 4-digit discord-tag. Requires `identify` scope                                                                         |
+| avatar        | ?string   | the user's [avatar hash](#DOCS_REFERENCE/image-formatting). Requires `identify` scope                                             |
+| bot?          | boolean   | whether the user belongs to an OAuth2 application. Requires `identify` scope                                                      |
+| system?       | boolean   | whether the user is an Official Discord System user (part of the urgent message system). Requires `identify` scope                |
+| mfa_enabled?  | boolean   | whether the user has two factor enabled on their account. Requires `identify` scope                                               |
+| banner?       | ?string   | the user's [banner hash](#DOCS_REFERENCE/image-formatting). Requires `identify` scope                                             |
+| accent_color? | ?integer  | the user's banner color encoded as an integer representation of hexadecimal color code. Requires `identify` scope                 |
+| locale?       | string    | the user's chosen [language option](#DOCS_REFERENCE/locales). Requires `identify` scope                                           |
+| verified?     | boolean   | whether the email on this account has been verified. Requires `email` scope                                                       |
+| email?        | ?string   | the user's email. Requires `email` scope                                                                                          |
+| flags?        | integer   | the [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account. Requires `identify` scope                           |
+| premium_type? | integer   | the [type of Nitro subscription](#DOCS_RESOURCES_USER/user-object-premium-types) on a user's account. Requires `identify` scope   |
+| public_flags? | integer   | the public [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account. Requires `identify` scope                    |
```

Comment:
This is just repetition of `Requires identify scope`. Could this be a text above the table and other scopes be a footnote?